lib/config/migrations/custom/package-rules-migration.ts

```diff
@@ -25,6 +26,29 @@ function renameKeys(packageRule: PackageRule): PackageRule {
   return newPackageRule;
 }
 
+function removeMatchLanguage(packageRule: PackageRule): PackageRule[] {
+  const newPackageRules: PackageRule[] = [];
+  const matchLanguages = packageRule.matchLanguages;
```

Comment:
so renaming is done before this migration?

Reply:
yes the renameMap seems to be applied before custom migrations.

Reply:
why don't we use a simple rename here too?

Reply:
🤔 probably not needed anymore, as I have removed the special migration logic for docker in https://github.com/renovatebot/renovate/pull/16534/commits/5ccf756a676fb3d6f72b4f14a6a932ff5669b573

I have removed it.